CHANGELOG.md

```diff
@@ -23,10 +23,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Add `asView` helper for getting SwiftUI views from StreamChatUI UIViewControllers (#1174)[https://github.com/GetStream/stream-chat-swift/pull/1174] 
 
 ### 🔄 Changed
+- Logic for displaying suggestions (commands or mentions) were not compatible with SwiftUI, so it's changed to AutoLayout (#1171)[https://github.com/GetStream/stream-chat-swift/pull/1171]
```

Comment:
the link format is broken (see preview)

Reply:
Someone broke the link format (possibly due to manual writing) and all of them broke due to copy/pastes..... Good catch, fixed them all

Reply:
All credits to @skorepak https://github.com/GetStream/stream-chat-swift/pull/1175#discussion_r649357018